include/rocksdb/advanced_options.h

```diff
@@ -1124,10 +1124,20 @@ struct AdvancedColumnFamilyOptions {
   // while set this flag to be `false`: user keys in the newly generated SST
   // files are of the same format as the existing SST files.
   //
+  // Currently only user comparator that formats user-defined timesamps as
+  // uint64_t via using one of the RocksDB provided comparator
+  // `ComparatorWithU64TsImpl` are supported.
+  //
   // When setting this flag to `false`, users should also call
   // `DB::IncreaseFullHistoryTsLow` to set a cutoff timestamp for flush. RocksDB
   // refrains from flushing a memtable with data still above
-  // the cutoff timestamp with best effort. Users can do user-defined
+  // the cutoff timestamp with best effort. If this cutoff timestamp is not set,
+  // flushing continues normally.
+  // NOTE: in order for the cutoff timestamp to work properly, users of this
+  // feature need to ensure to write to a column family with globally
+  // non-decreasing user-defined timestamps.
```

Comment:
AFAIK this is always required for user-defined timestamp

Reply:
Thank you for calling out this! This is intended to describe a more strict requirement in order to use this feature. If users are just using the UDT feature, the requirement is for the same user key without timestamp, the user-defined timestamps should be non-decreasing. This feature requires the newest UDT in a newer Memtable should be newer than that for an older memtable, across different user keys, which is a bit nuanced, but shouldn't be surprising for users that are using this feature.